packages/webpack/src/config/base.js

```diff
@@ -195,8 +195,12 @@ export default class WebpackBaseConfig {
   }
 
   resolve () {
-    // Prioritize nested node_modules in webpack search path (#2558)
-    const webpackModulesDir = ['node_modules'].concat(this.buildContext.options.modulesDir)
+    const webpackModulesDir = [
+      // Prioritize nested node_modules in webpack search path (#2558)
+      'node_modules',
+      // Ensure that loaders defined in @nuxt/webpack's package.json can be found
+      path.join(path.dirname(require.resolve('@nuxt/webpack/package.json')), 'node_modules')
```

Comment:
- As this core is already inside `@nuxt/webpack/dist`, we can simply use `path.resolve(__dirname, '../node_modules)`
- I think would be better adding it **after** modulesDir as a fallback

Reply:
I used `path.join(path.dirname())` so that this line of code does not depend on how/where the JS file is built

Reply:
I see, this line break the tests... I will change accordingly